src/Domain/Model/Donor/Address/NoAddress.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\DonationContext\Domain\Model\Donor\Address;
+
+use WMDE\Fundraising\DonationContext\Domain\Model\Address;
+
+/**
+ * @license GPL-2.0-or-later
+ */
+class NoAddress implements Address {
+
+	private string $streetAddress = '';
+	private string $postalCode = '';
+	private string $city = '';
+	private string $countryCode = '';
```

Comment:
I'm assuming that these values should never change for what this class is supposed to do. Would it make sense to turn those fields into constants to emphasise that fact? or rather unnecessary?
```suggestion
	private const string $streetAddress = '';
	private const string $postalCode = '';
	private const string $city = '';
	private const string $countryCode = '';
```

Reply:
This is not how things work in PHP, but thanks for pointing it out! This is the result of some unfortunate copy-pasting and I'll change it.

Reply:
that's even smarter now :+1: